inductiva/molecules/scenarios/md_water_box/post_processing.py

```diff
@@ -0,0 +1,39 @@
+"Postprocessing steps for the MDWaterBox scenario."
+import os
+import MDAnalysis as mda
+import nglview as nv
+from pathlib import Path
+
+
+class MDWaterBoxSimulationOutput:
```

Comment:
Yes, you can decrease the name. I even want to discuss further if this is the right usage inside the scenario, but we can all talk tomorrow about it.